src/Training.AirBnb.Clone.Backend/AirBnB.Infrastructure/Ratings/Services/RatingRecalculationService.cs

```diff
@@ -0,0 +1,122 @@
+using AirBnB.Application.Ratings.Services;
+using AirBnB.Domain.Constants;
+using AirBnB.Domain.Entities;
+using AirBnB.Persistence.Caching.Brokers;
+using AirBnB.Persistence.DataContexts;
+using Microsoft.EntityFrameworkCore;
+
+namespace AirBnB.Infrastructure.Ratings.Services;
+
+public class RatingRecalculationService(ICacheBroker cacheBroker, AppDbContext appDbContext) 
```

Comment:
I think this must be `RatingProcessingService` as we don't have anything crossing multiple domains here